src/test/resources/test-application-rdbms1.properties

```diff
@@ -126,7 +126,7 @@ klaw.saas.ssl.pubkey=C:/Software/confluent-5.3.1-2.12/certs/Klaw_PublicKey.zip
 klaw.saas.ssl.clientcerts.location=C:/Software/confluent-5.3.1-2.12/certs/clientcerts
 klaw.saas.ssl.clusterapi.truststore=C:/Software/confluent-5.3.1-2.12/certs/clientcerts/client.truststore.jks
```

Comment:
It seems not related to this PR however it seems it should be fixed: location should be a relative path rather than absolute

Reply:
Good catch!

Reply:
Good one. Updated, and also removed some google captcha settings